src/org/zaproxy/zap/extension/alertFilters/AlertFilterAPI.java

```diff
@@ -66,15 +66,23 @@ public AlertFilterAPI(ExtensionAlertFilters extension) {
 		super();
 		this.extension = extension;
 
-		this.addApiView(new ApiView(VIEW_ALERT_FILTER_LIST, new String[] { PARAM_CONTEXT_ID }));
+		ApiView alertFilterList = new ApiView(VIEW_ALERT_FILTER_LIST, new String[] { PARAM_CONTEXT_ID });
+		alertFilterList.setDescriptionTag("alertFilters.api.view.alertFilterList");
```

Comment:
Are the setDescriptionTag calls needed? API endpoint description tags are automatically generated...

Reply:
Yes, but in this case with the wrong prefix (`alertFilter` instead of `alertFilters`, which is the extension's i18n prefix). I can change the i18n prefix/extension package to be the same as the API (that might break something though).

Reply:
No thats fine, just checking :)